code/game/objects/machinery/research.dm

```diff
@@ -107,6 +107,8 @@
 /obj/machinery/researchcomp/ui_static_data(mob/user)
 	. = ..()
 
+	var/points = GLOB.round_statistics.points_from_research
+	.["acquired_points"] = points
```

Comment:
```suggestion
	.["acquired_points"] = GLOB.round_statistics.points_from_research
```

Reply:
For some reason this didn't work when I was testing it, but works now...